Gemfile

```diff
@@ -14,7 +14,7 @@ group :test do
   gem 'factory_bot_rails', '~> 5.0'
   gem 'faker', '~> 2.2'
   gem 'rake', '~> 12.3'
-  gem 'capybara', '~> 2.3.0'
+  gem 'capybara', '~> 3.33.0'
```

Comment:
Some of the headless drivers were introduced in version 3. Only a few tests broke after the upgrade.